src/app/components/impersonification-notification/impersonification-notification.html

```diff
@@ -0,0 +1,22 @@
+<div class="container" ng-show="requests.length > 0">
```

Comment:
There should only be one `.container` in a bootstrap layout. There is already one in `layout.html`, so you can remove this one.